packages/perseus/src/widgets/__testdata__/explanation.testdata.ts

```diff
@@ -125,3 +125,46 @@ export const ipsumExample: PerseusRenderer = {
         },
     },
 };
+
+export const wideButton: PerseusRenderer = {
+    content: `Unidentified vessel travelling at sub warp speed, bearing 235.7. 
+                Fluctuations in energy readings from it, Captain. 
+                All transporters off. 
+                A strange set-up, but I'd say the graviton generator is depolarized. 
+                The dark colourings of the scrapes are the leavings of natural rubber, 
+                    a type of non-conductive sole used by researchers experimenting with electricity. 
+                The molecules must have been partly de-phased by the anyon beam.
+                \n[[\u2603 explanation 1]]\n\nSensors indicate no shuttle or other ships in this sector.
+                According to coordinates, we have travelled 7,000 light years and are located near [the system J-25](#).
+                Tractor beam released, sir. Force field maintaining our hull integrity.
+                Damage report? Sections 27, 28 and 29 on decks four, five and six destroyed.
```

Comment:
excellent test data

Reply:
Thank you! I think everyone should use this kind of [filler content](https://vlad-saling.github.io/star-trek-ipsum/) for their examples. (I might be biased, though.)